general/server/plugins/add-ons.md

```diff
@@ -0,0 +1,86 @@
+
+**Link:**
+
+* [Github](https://github.com/jasongdove/ErsatzTV)
+
+### JF-Secure
+
+#### Description
+A turn-key reverse proxy tool that makes use of Caddy. It will setup a reverse proxy for Jellyfin with the goal of being easy to use, even for novice users.
+
+**NOTE: THIS IS WINDOWS ONLY**
+
+**Link:**
+
+* [Github](https://github.com/dab2020/JF-secure)
+* [Latest Release](https://github.com/dab2020/JF-secure/releases)
+
+### Jellyfin_HA
+
+#### Description
+For all you smart home Jellyfin users, Jellyfin HA enables you to use your media server with Home Assistant integration.
+
+**Link:**
+
+* [Github](https://github.com/koying/jellyfin_ha)
+* [Latest Release](https://github.com/koying/jellyfin_ha/releases)
+
+### Jellyfin Party Extensions
+
+#### Description
+Have you ever wanted to have all your friends watch a movie or tv show together without having to give them accss to the Jelyfin UI. Well here it is. Just pull up a chrome make a link start the movie and stream with all your friends through your chrome browser to theirs.
+
+**Links:**
+
+* [GitHub](https://github.com/Jelly-Party/jelly-party-extension)
+* [Jellyfin Chrome Extensions Install](https://www.jelly-party.com/)
+
+### Jellyfin Helm
+
+#### Description
+For all you chart trackers and multiple web app users. This is a way to add all your jellyfin stats to your helm web app.
+
+* [Github](https://github.com/brianmcarey/jellyfin-helm)
+
+
+### JFA-Go
+
+#### Description
+Have you wanted your users to be able to remotely reset their password without you having to do it. Well JFA-Go is it. All you have to do is set it up, link your jellyfin install, and profit. This not only lets users reset their password, but you can add new accounts set a static layout for the main pages libraries on all users from a dummy account with permissions. 
+
+**Links:**
+
+* [GitHub](https://github.com/hrfee/jfa-go)
+* [Latest Release](https://github.com/hrfee/jfa-go/releases)
+
+### TV Intro Detection
+
+#### Description
+This is an automated way to skip all your TV Show introductions like netflix does. Unfortuntly it does not do outros yet, but its a great start.
+
+**NOTE: DOCKER ONLY**
+
+**Links:**
+
+* [GitHub](https://github.com/jellyfin/jellyfin-plugin-emailnotifications)
+* [Documentation](https://mueslimak3r.github.io/tv-intro-detection/)
+
+### Emby2Jelly
+
+#### Description
+This is a good way to move jellyfin from one server to another host of jellyfin, or move from Emby to Jellyfin without losing data metadata critical data.
```

Comment:
```suggestion
This add-on provides a way to migrate users and watched content from Emby to Jellyfin.
```
They only state that it works when migrating from Emby to Jellyfin. Did you test this with moving Jellyfin as well? 

Also, they state that the Jellyfin version tested is 10.5.4 which is quite old already. Is this tested on 10.8.x?